app/ISC_DHCP_daemon/doc/README.head.md

```diff
@@ -0,0 +1,8 @@
+Basic monitoring of the [ISC DHCP daemon](https://www.isc.org/downloads/dhcp/).
+
+## Usage
```

Comment:
Consider moving the usage section to the template.

Reply:
Removed and updated with the help of the new default usage generation (#74).